lib/extensions/postgres_cdc_rls/subscriptions_checker.ex

```diff
@@ -139,14 +142,13 @@ defmodule Extensions.PostgresCdcRls.SubscriptionsChecker do
     Enum.reduce(pids, [], fn pid, acc ->
       case :ets.lookup(tid, pid) do
         [] ->
-          Logger.error("Can't find pid in subscribers table: #{inspect(pid)}")
+          Telemetry.execute([:realtime, :subscriptions_checker, :pid_not_found], 1, %{})
```

Comment:
Do you need to tag by tenant?

Reply:
we don't have access to them here it would require a bigger refactor so I will keep it as a single number that we can measure for now

Reply:
If there's an urgent need for the tenant and you prefer not to pass the ref via a function, it can be obtained through `Logger.metadata()[:project]`

Reply:
saw that id was actually the tenant id, used that instead and moved things so we have this as a tenant metric